launch/launch/actions/timer_action.py

```diff
@@ -78,22 +78,22 @@ def __init__(
                 stacklevel=2)
         self.__period = type_utils.normalize_typed_substitution(period, float)
         self.__actions = actions
-        self.__context_locals = {}  # type: Dict[Text, Any]
-        self.__completed_future = None  # type: Optional[asyncio.Future]
+        self.__context_locals: Dict[Text, Any] = {}
+        self.completed_future: Optional[asyncio.Future] = None
         self.__canceled = False
-        self.__canceled_future = None  # type: Optional[asyncio.Future]
+        self.canceled_future: Optional[asyncio.Future] = None
         self.__cancel_on_shutdown = type_utils.normalize_typed_substitution(
             cancel_on_shutdown, bool)
         self.__logger = launch.logging.get_logger(__name__)
 
-    async def __wait_to_fire_event(self, context):
+    async def wait_to_fire_event(self, context):
```

Comment:
Others might have a different opinion, but I think we should keep a single underscore prefix for this method name (as well as `completed_future` and `canceled_future`) to indicate that this is *not* intended to be user-facing API.

Reply:
Makes sense to me, I made the change.